src/Sylius/Bundle/AdminBundle/Resources/config/grids/customer_order.yml

```diff
@@ -8,6 +8,10 @@ sylius_grid:
                         method: createByCustomerIdQueryBuilder
                         arguments:
                             customerId: $id
+            fields:
+                customer:
+                    type: string
```

Comment:
This line is not needed, I suppose

Reply:
without this line I get error :(
![Screenshot 2019-09-03 at 08 38 38](https://user-images.githubusercontent.com/29897151/64149471-41d3f600-ce26-11e9-95ab-ddb407ee5f75.png)